### ISSUE TITLE
a11y: "Get this Book" button should have aria-role="dialog"

### ISSUE THREAD

**Issue by brittweinstein** [Connexions/webview#1759]
@brittweinstein commented on [Tue Jul 24 2018](https://github.com/Connexions/content-managers/issues/129)

From https://drive.google.com/file/d/0BxiTbl6K_ZNRelpxRlkwYkZnb3c/view

Issue 12: "Get this book" button has inaccessible pop-up window

Reference to standards
1.3.1 Info and Relationships: Information, structure, and relationships conveyed through presentation can be
programmatically determined or are available in text. (Level A)
4.1.2 Name, Role, Value: For all user interface components (including but not limited to: form elements, links and
components generated by scripts), the name and role can be programmatically determined; states, properties, and
values that can be set by the user can be programmatically set; and notification of changes to these items is
available to user agents, including assistive technologies. (Level A)
2.4.7 Focus Visible: Any keyboard operable user interface has a mode of operation where the keyboard focus
indicator is visible. (Level AA)
Suggestion
It is better to make a pop-up window using ARIA role=dialog

In general, the approach was correct, but we need to change some attributes and remove the color piece, and fix spelling. - Phil S. and Kerwin

https://github.com/Connexions/webview/pull/1624![Screen Shot 2018-07-24 at 4.00.33 PM.png](https://images.zenhubusercontent.com/5a01faba8a75884b90866af7/363134f9-d36b-4b9d-a9a0-e23874ea6bd4)

**Comment by Stackblocks** [comment]
Displaying as specified with updated version of content / code.
 ![image.png](https://images.zenhubusercontent.com/5af1f4c24b5806bc2bd257f2/b900f22e-34fd-420c-bfbf-0e1c5bd422de)
